Sco andrew no rain

Andrew-Libby opened the pull request.

SynixeBrodsky reviewed (changes requested):
Found issues with the following files:
* contracts/SCO30_Andrew_NoRain.Desert_E/edit_me/briefing/situation.html
  * Situation briefing is too long for Discord

BrettMayson commented on contracts/SCO30_Andrew_NoRain.Desert_E/edit_me/briefing/objectives.html:
```diff
@@ -0,0 +1,5 @@
+<font color='#E3D310'>PRIMARY</font>
+Find Abdul Akar and capture him.
```
```suggestion
- Find and Cpature Abdul Akar
```

Andrew-Libby replied:
Done.

BrettMayson commented on contracts/SCO30_Andrew_NoRain.Desert_E/edit_me/briefing/objectives.html:
```diff
@@ -0,0 +1,5 @@
+<font color='#E3D310'>PRIMARY</font>
+Find Abdul Akar and capture him.
+
+<font color='#595305'>SECONDARY</font>
+Kill Abdul Akar if you are unable to successfully capture him and extract his body.
```
```suggestion
<font color='#595305'>SECONDARY</font>
- Extract Abdul Akar's body if he refuses capture
```

Andrew-Libby replied:
Done.

BrettMayson reviewed (changes requested):
- There are way too many lights at base
- I would put something by the vehicle spawn so people know where it is and not to stand there

Andrew-Libby commented:
> * There are way too many lights at base
> * I would put something by the vehicle spawn so people know where it is and not to stand there

Reduced lights at spawnpoint. 
Added parking sign by vehicle spawn.
Adjusted the briefing points that were suggested.
